src/WebUI/Pods/PodsTable.tsx

```diff
@@ -26,21 +27,36 @@ export interface IPodsTableProperties extends IVssComponentProperties {
     podsToRender: V1Pod[];
     headingText?: string;
     nameFilter?: string;
+    showWorkloads?: boolean;
 }
 
 export class PodsTable extends BaseComponent<IPodsTableProperties> {
+    private statusCount: { [key: string]: number } = {};
```

Comment:
generally we use _statusCount as private and also keep this at the end of the class